### ISSUE TITLE
Cloud: Vmware add option to rewrite disk with zeros during provisioning

### ISSUE THREAD

**Issue by Shinzu** [saltstack/salt#44040]
### Description of Issue/Question
we want the option to eagerly scrub(rewrite with zeros) the disk during provisioning 
it should be set in a profile

```
linux-base:
  ...
  devices:
      disk:
        Hard disk 1:
          size: 40
          controller: 'SCSI controller 1'
          eagerly_scrub: True
```

**Comment by Ch3LL** [comment]
Seems reasonable enough. will approve as a feature request. ping @saltstack/team-cloud

**Comment by gtmanfred** [comment]
This seems reasonable, looks like there is already a PR for this as well, so thanks!
Daniel

**Comment by Ch3LL** [comment]
Oh I didn't catch the PR. Thanks !!!!!!!!

**Comment by Shinzu** [comment]
i closed the previous 2 pull request , forgot to sign them , sry for that

with the last pull request all should be fine

**Comment by rallytime** [comment]
Closed with #44061 

Thank you for fixing that @Shinzu and congratulations on your first Salt PR :)
